__tests__/app.test.js

```diff
@@ -162,4 +153,59 @@ describe("GET /api/articles/:article_id/comments", () => {
             expect(body.msg).toBe("Bad Request")
         })
     })
+})
+
+describe("POST /api/articles/:article_id/comments", () => {
+    it("Should return a status code: 201 and the posted comment", () => {
+        const newComment = { username: 'butter_bridge', body: 'Im not Kakarot?'}
+        
+        return request(app)
+             .post("/api/articles/1/comments")
+             .send(newComment)
+             .expect(201)
+             .then(({ body }) => {
+                const { comment } = body
+                
+                expect(comment).toEqual({
+                    comment_id: expect.any(Number),
+                    body: expect.any(String),
+                    votes: expect.any(Number),
+                    author: expect.any(String),
+                    article_id: expect.any(Number),
+                    created_at: expect.any(String)
+                })
+             })
+    })
```

Comment:
Nice first test, but here we can be more specific. The author and body we should know and should be testing specifically here to ensure our comment has actually been added.